### PR TITLE
Book

### PR DESCRIPTION
**Pull request by Paradoxallist**
implement entity book

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	<name>bookshop</name>
 	<description>Demo project for Spring Boot</description>
 	<properties>
-		<java.version>21</java.version>
+		<java.version>17</java.version>
```
Comment (Vladyslav-Kuksiuk) [inline pom.xml:17]: 123

File: src/main/java/store/bookshop/model/Book.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```
Comment (Rommelua) [inline src/main/java/store/bookshop/model/Book.java]: Do not use Data with entity class.
Add Getter, Setter, etc.

File: src/main/java/store/bookshop/model/Book.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    public Book(String title, String author, String isbn, BigDecimal price) {
+        this.title = title;
+        this.author = author;
+        this.isbn = isbn;
+        this.price = price;
+    }
+
+    public Book() {
+    }
```
Comment (Rommelua) [inline src/main/java/store/bookshop/model/Book.java]: use lombok for constructors

File: src/main/java/store/bookshop/model/Book.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Nonnull
+    private String title;
+    @Nonnull
+    private String author;
+    @Nonnull
+    @Column(unique = true)
+    private String isbn;
```
Comment (Rommelua) [inline src/main/java/store/bookshop/model/Book.java]:
```suggestion
    @Column(name = "title", nullable = false)
    private String title;
    @Column(name = "author", nullable = false)
    private String author;
    @Column(name = "isbn", nullable = false, unique = true)
    private String isbn;
```

**Review by Rommelua — CHANGES_REQUESTED** [review]

**Review by koval-kn — APPROVED** [review]

**Pull request merged**